core/chains/evm/txmgr/models.go

```diff
@@ -41,7 +41,7 @@ type EthTxMeta struct {
 	SubID *uint64 `json:"SubId,omitempty"`
 
 	// Used for keepers
-	UpkeepID *int64 `json:"UpkeepID,omitempty"`
+	UpkeepID *string `json:"UpkeepID,omitempty"`
```

Comment:
why not bigInt here

Reply:
this is the hex-encoded upkeep ID, ex `UPx1234...ABCD`